src/testing/builderbot/simulator/test_builderbot.argos.in

```diff
@@ -129,7 +130,7 @@
   <!-- * Visualization * -->
   <!-- ****************** -->
   <visualization>
-    <qt-opengl lua_editor="false" show_boundary="false">
+    <qt-opengl lua_editor="true" show_boundary="false">
```

Comment:
Please always double check that you are not introducing changes like this in your commits